src/utils/artifact/generateAragonArtifact.ts

```diff
@@ -26,16 +30,19 @@ function _generateAragonArtifact(
     ...arapp,
 
     // Artifact appears to require the abi of each function
-    functions: functions.map(parsedFn => ({
-      roles: parsedFn.roles.map(role => role.id),
-      notice: parsedFn.notice,
-      abi:
-        abiBySignature[parsedFn.sig] ||
-        (parsedFn.sig === 'fallback' ? abiFallback : null),
-      // #### Todo: Is the signature actually necessary?
-      // > Will keep them for know just in case, they are found in current release
-      sig: parsedFn.sig
-    })),
+    functions: functions.map(
+      parsedFn =>
+        ({
+          roles: parsedFn.roles.map(role => role.id),
+          notice: parsedFn.notice,
+          abi:
+            abiBySignature[parsedFn.sig] ||
+            (parsedFn.sig === 'fallback' ? abiFallback : null),
+          // #### Todo: Is the signature actually necessary?
+          // > Will keep them for know just in case, they are found in current release
+          sig: parsedFn.sig
+        } as AragonArtifactFunction)
```

Comment:
I needed to include the `AragonArtifactFunction` for the return function otherwise we were getting a type error during build.